tests/phpunit/tests/rest-api/rest-schema-setup.php

```diff
@@ -368,19 +368,6 @@ public function test_build_wp_api_client_fixtures() {
 			);
 			$this->assertTrue( ! empty( $data ), $route['name'] . ' route should return data.' );
 
-			if ( version_compare( PHP_VERSION, '5.4', '>=' ) ) {
-				$fixture = $this->normalize_fixture( $data, $route['name'] );
```

Comment:
Same as above: we should always run the code inside this block, rather than deleting it.

Reply:
done with latest commit and the one on new L377-382 I put back, but removed the PHP version condition, because it needs to check for single site only.